readme.md

```diff
@@ -17,6 +17,17 @@ Using jQuery, write code that makes an AJAX get request to "http://kittengifs.co
 
 Your Answer:
 ```js
+var url = "http://kittengifs.com/gifs"
+$.ajax({
+  dataType: 'json',
+  url: url,
+  data: data
+  console.log(data)
```

Comment:
You added this `console.log` command inside an object that is being passed as an argument to the ajax method.  This should just contain key/value pairs.